site/content/docs/5.3/getting-started/accessibility.md

```diff
@@ -78,6 +78,10 @@ Each component then specifies its own visible focus style when needed, for examp
 
 #### Under a fixed header
 
+{{< callout warning >}}
```

Comment:
I think this callout warning places just before the code example would be more understandable...

Reply:
I don't have a strong opinion on this. Trying your idea!

Reply:
Discussed in daily and this last place was agreed on.